File: Leetcode/389-Find-the-Difference/FindTheDifference.go
func findTheDifference(s string, t string) byte {
	cur := t[0]
	for i := 1; i < len(t); i++ {
		cur = cur ^ t[i]
	}
	for i := 0; i < len(s); i++ {
		cur = cur ^ s[i]
	}
	return cur
}

File: Leetcode/389-Find-the-Difference/FindTheDifference.py
class Solution(object):
    def findTheDifference(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: str
        """
        cur = ord(t[0])
        for i in range(1, len(t)):
            cur = cur^ord(t[i])
        for c in s:
            cur = cur^ord(c)
        return chr(cur)
    
